docs/includes/dev-kit-stop-debugging.md

```diff
@@ -0,0 +1,7 @@
+## Stop testing your Office Add-in
+
+Once you are finished testing and debugging the add-in, close the add-in by following these steps.
+
+1. Open the extension by selecting the Office Add-ins Development Kit icon in the **Activity Bar**.
+1. Select **Stop Previewing Your Office Add-in**. This closes the web server and removes the add-in from the registry and cache.
+1. Close the Office application window at your convenience.
```

Comment:
They should not leave it open because any buttons connected with the add-in will still be visible, but they will no longer work.

```suggestion
1. Close the Office application window.
```